lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -140,11 +140,11 @@ class UnusedBytes extends Audit {
       rawValue: wastedMs,
       score: UnusedBytes.scoreForWastedMs(wastedMs),
       extendedInfo: {
-        value: {
+        value: Object.assign({
           wastedMs,
           wastedKb,
           results,
-        },
+        }, result.extendedInfo),
```

Comment:
maybe add a comment that this merges in any `extendedInfo` provided by the derived audit?

Reply:
done